packages/lib/server/repository/organization.ts

```diff
@@ -305,6 +305,13 @@ export class OrganizationRepository {
           },
         },
       },
+      select: {
+        parentId: true,
+        id: true,
+        name: true,
+        logoUrl: true,
+        slug: true,
+      },
```

Comment:
confirmed that these fields are all we need for the instances that call this function